src/types.rs

```diff
@@ -13,6 +13,11 @@ use time::{Duration, OffsetDateTime};
 
 use crate::Error;
 
+/// The default number of seconds that it takes for a Google Cloud auth token to expire.
+/// This appears to be the default from practical testing, but we have not found evidence
+/// that this will always be the default duration.
+pub(crate) const DEFAULT_TOKEN_DURATION: Duration = Duration::seconds(3600);
```

Comment:
Sorry -- but this `const` only gets used in `gcloud_authorized_user`, so I think it should get moved there, too.

Reply:
No need to apologize; That was an oversight on my part. Thanks for pointing it out!